### PR TITLE
re-render for conf 2023

### PR DESCRIPTION
**Pull request by simonpcouch**
Taking a look through right now to make sure everything looks fine, another set of eyes definitely appreciated. :)

### CODE REVIEW

**Comment by simonpcouch** [comment]
Still had some dev package versions--re-rendering.

**Comment by hfrick** [comment]
I'll look at intro: 1, 3, 5 (and any other if you have some to assign)

**Comment by simonpcouch** [comment]
Ah, I see why you saw differences in versions in #288. The intro "Our versions" code defines `deps` but then uses `pkgs`, from earlier in the slides, when printing.

https://github.com/tidymodels/workshops/blob/0c8c7b679a746d73452606f46c2aeb660d6672ae/slides/advanced-01-introduction.qmd#L267

https://github.com/tidymodels/workshops/blob/0c8c7b679a746d73452606f46c2aeb660d6672ae/slides/advanced-01-introduction.qmd#L282-L285

File: docs/slides/figures/cubist-autoplot-1.svg
Comment (EmilHvitfeldt) [inline docs/slides/figures/cubist-autoplot-1.svg:1]: this one shows different results. but not enough to change interpretation

File: docs/slides/figures/stack-autoplot-1.svg
Comment (EmilHvitfeldt) [inline docs/slides/figures/stack-autoplot-1.svg:1]: this one is quite different

File: docs/slides/extras-vetiver.html [hunk 1]
```diff
@@ -442,7 +442,7 @@ <h2>Deploy your model <img data-src="hexes/vetiver.png" class="absolute" style="
 <span id="cb2-8"><a href="#cb2-8"></a><span class="co">#&gt; ├──[cookieParser]</span></span>
 <span id="cb2-9"><a href="#cb2-9"></a><span class="co">#&gt; ├──[sharedSecret]</span></span>
 <span id="cb2-10"><a href="#cb2-10"></a><span class="co">#&gt; ├──/logo</span></span>
-<span id="cb2-11"><a href="#cb2-11"></a><span class="co">#&gt; │  │ # Plumber static router serving from directory: /Users/emilhvitfeldt/Library/R/arm64/4.3/library/vetiver</span></span>
+<span id="cb2-11"><a href="#cb2-11"></a><span class="co">#&gt; │  │ # Plumber static router serving from directory: /Users/simoncouch/Library/R/arm64/4.2/library/vetiver</span></span>
```
Comment (EmilHvitfeldt) [inline docs/slides/extras-vetiver.html:445]: fairly minor. But you rendered using R 4.2 and I did it last time with 4.3.

**Review by EmilHvitfeldt — APPROVED** [review]
Few notes!

Should we rerender with 4.3? the latest version is technically 4.3.1

**Comment by simonpcouch** [comment]
All looks good to me besides slides 19 and 20 in the iterative search slides. I don't see the images show up, though I've had this same issue locally before. If others don't see this, thumbs up from me on merging.

File: docs/slides/figures/stack-autoplot-1.svg
Comment (simonpcouch) [reply docs/slides/figures/stack-autoplot-1.svg:1]: Yeah. I wonder if there's a missing `set.seed()` somewhere in there.

**Comment by simonpcouch** [comment]
Not particularly concerned with the R version difference but all good if you think that's a needed change!

**Comment by EmilHvitfeldt** [comment]
Do you also see this
Screenshot 2023-09-15 at 1 52 46 PM

**Comment by simonpcouch** [comment]
I do. Switching to 9 in that lower bound fixes for me:

Screenshot 2023-09-15 at 1 59 08 PM

Since that file hadn't been changed yet in this PR, I'll go ahead and push that fix.

File: docs/slides/figures/stack-autoplot-1.svg
Comment (EmilHvitfeldt) [reply docs/slides/figures/stack-autoplot-1.svg:1]: I'm getting this locally:

Screenshot 2023-09-15 at 2 13 03 PM

**Pull request merged**